<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacherform</class>
 <widget class="QWidget" name="teacherform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/picture/QQ截图20170708141021.png);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>密码修改</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;旧密码：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;新密码：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;确认密码：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="oldEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="newEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="confireEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancleButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>270</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>试题管理</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;类型：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="typecomboBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C++</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Java</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C#</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;题目列表：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>301</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;测试题：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QListView" name="listView_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>301</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>110</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>170</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>220</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>320</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;本题答案：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="answercomboBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>320</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>成绩管理</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>491</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;学生学号：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="numberEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
